### ISSUE TITLE
Synthesis UI: feedback

### ISSUE THREAD

**Issue by paulrouget** [mozilla/browser.html#622]
I dropped my regular browser and only use browser.html now. It's been a week so far. It's pretty stable now.

As I did for the initial design (https://github.com/mozilla/browser.html/issues/288), here is my feedback for the new synthesis UI.

My main concern is the lack of regular tabs.

If I casually browse the web, let's say, reddit or hackernews, I can deal with the previews. I basically don't care about tabs and exhaust the tabs until there's none.

But for many tasks, like, work (github & bugzilla pages), buying things online (amazon links), documentation (MDN) or visiting dxr,mozilla.org, the preview system is frustrating and inefficient:
- I can only see 3 or 4 tabs max, where as with a regular browser, I see at least 8 tabs (for the same window size, 3.5 tabs with browser.html, 8 with Safari & Firefox, ~12 with Chrome).
- Most of the time, finding the right tab is hard because I see a lot of confusing (https://github.com/mozilla/browser.html/issues/610) and duplicate previews (I then use the title, which makes 90% of the card useless)
- Getting to the right tab is slow. Before, I was spotting the tab to go to with my eyes and then clicked/ctrl-tabed to it. Now I have to get to preview mode first and search for it, which is slow and hide my current page.
- Often I don't want to select a new tab, but just have a quick look at what I have open. Knowing what is open used to be a very simple task, now it's not. I'm basically losing context every time I want to know what I have open.
- I really need a back button, add least to know if I can go back. Also, see https://github.com/mozilla/browser.html/issues/612 about discoverability
- tabs in RTL feels backward. See https://github.com/mozilla/browser.html/issues/621
- I'd prefer if we'd use the regular stoplight buttons on mac: https://github.com/mozilla/browser.html/issues/583

Basically, I believe that if one uses more than 3 or 4 tabs, the current design doesn't work. It's too hard to keep track of your session.

@gordonbrander what do you think?

**Comment by gordonbrander** [comment]
Addressed through design round 3 and current work in master.
